### PR TITLE
Data adder for data pack

### PR DESCRIPTION
**Pull request by hepengfe**
This PR fixes https://github.com/asyml/forte/issues/895

### Description of changes
* function `add_text` to add text into DataPack
* function `add_audio` to add audio into DataPack

### Possible influences of this PR.
Describe what are the possible side-effects of the code change.

### Test Conducted
Describe what test cases are included for the PR.

### CODE REVIEW

**Review by mylibrar — COMMENTED** [review]
So where are the `add_audio` and `add_text` functions?

**Comment by hepengfe** [comment]
> So where are the `add_audio` and `add_text` functions?

I deleted in one commit accidentally. I have added it back.

File: forte/data/data_pack.py [hunk 1]
```diff
@@ -639,6 +654,19 @@ def set_audio(
         ap.set_cache(audio)
         ap.sample_rate = sample_rate
 
+    def add_audio(self, audio):
```
Comment (mylibrar) [inline forte/data/data_pack.py:657]: Let's also add a warning message in `set_audio` that encourages users to use `add_audio` instead. https://github.com/asyml/forte/pull/891#discussion_r942677425

File: docs/spelling_wordlist.txt [hunk 2]
```diff
@@ -157,3 +157,4 @@ PathLike
 ElementType
 customizable
 tokenizes
+param
```
Comment (mylibrar) [inline docs/spelling_wordlist.txt:160]: I don't see `param` being used anywhere in the PR. Is there any reason to add it to the wordlist?

**Review by mylibrar — APPROVED** [review]

**Pull request merged**